lib/jazzy/config.rb

```diff
@@ -220,6 +220,11 @@ def expand_path(path)
         SourceDeclaration::AccessControlLevel.from_human_string(acl)
       end
 
+    config_attr :lint,
+      command_line: '--[no-]lint',
+      description: "Generate linter output.",
+      default: false
```

Comment:
Based on the flags I'd suggested we'd have something like the following:

> always build docs but send notification with lint output iff lint fails

    jazzy --lint --dry-run || { notify }
    jazzy

> notify and do not build docs if lint fails

    jazzy --lint --dry-run || { notify; exit }
    jazzy

> no lint; just build docs

    jazzy